package.json

```diff
@@ -78,6 +78,16 @@
                 "title": "Lingua Franca: Build and Run"
             }
         ],
+        "configuration": {
+            "title": "LinguaFranca",
+            "properties": {
+                "linguafranca.partialBuildOnSave": {
```

Comment:
I went with `generateCodeOnSave` in the end because we don't actually do a complete build on save (i.e., we don't compile the generated code). I hope that makes sense.

Reply:
Yes, that makes perfect sense!